phone_validation_e164/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+{
+    'name': "Nitrokey Customization",
+    'description': """
+    We have added defaule e164 phone formator
+    """,
+    'category': 'Nitrokey',
+    'version': '12.0.1.0.0',
+    'depends': [
+        'phone_validation'
```

Comment:
Below phone_validation_mixin is imported. Is it required as a dependency too?

Reply:
Yes, it is required as we have utilized the phone validation mixin(phone.validation.mixin) model